dist/helm/templates/deployment.yaml

```diff
@@ -1,11 +1,12 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ template "oci-registry.fullname" . }}
+  name:  {{ template "oci-registry.fullname" . }}
   labels:
     {{- include "oci-registry.labels" . | nindent 4 }}
+    helm.sh/chart: {{ template "oci-registry.chart" . }}
```

Comment:
You undid one of your changes :)

No problem though - when I get done my dayjob today I'll merge, fix this, and cut a release

Reply:
thanks for noticing, fixed now